src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IDynamicMemberReferenceExpression.vb

```diff
@@ -0,0 +1,370 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.UnitTests.Semantics
+
+    Partial Public Class IOperationTests
+        Inherits SemanticModelTestBase
+        <Fact()>
+        Public Sub IDynamicMemberReferenceExpression_SimplePropertyAccess()
+            Dim source = <![CDATA[
+Option Strict Off
+Module Program
+    Sub Main(args As String())
+        Dim d = Nothing
+        d.Foo'BIND:"d.Foo"
+    End Sub
+End Module
+]]>.Value
+
+            Dim expectedOperationTree = <![CDATA[
+IOperation:  (OperationKind.None) (Syntax: 'd.Foo')
+  Children(1):
+      IDynamicMemberReferenceExpression (Member name: Foo, Containing Type: Object) (OperationKind.DynamicAccessExpression, Type: System.Object) (Syntax: 'd.Foo')
+        Instance Receiver: ILocalReferenceExpression: d (OperationKind.LocalReferenceExpression, Type: System.Object) (Syntax: 'd')
```

Comment:
After @jinujoseph recent change, we should ensure that null fields and empty arrays are still included in the dumper output. So the output here should include both ContainingType and TypeArguments.